packages/vike-react-redux/package.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "vike-react-redux",
+    "version": "0.0.0",
+    "type": "module",
+    "exports": {
+      "./config": "./dist/config.js",
+      "./__internal/onAfterRenderHtml": "./dist/onAfterRenderHtml.js",
+      "./__internal/onBeforeRenderHtml": "./dist/onBeforeRenderHtml.js",
+      "./__internal/onBeforeRenderClient": "./dist/onBeforeRenderClient.js",
+      "./__internal/Wrapper": "./dist/Wrapper.js"
+    },
+    "scripts": {
+      "dev": "tsc --watch",
+      "build": "rimraf dist/ && tsc",
+      "release": "release-me patch",
+      "release:minor": "release-me minor",
+      "release:major": "release-me major",
+      "release:commit": "release-me commit"
+    },
+    "peerDependencies": {
+      "@reduxjs/toolkit": ">=2",
+      "react-redux": ">=9",
```

Comment:
How about we define `react-redux` and `@reduxjs/toolkit` as both `dependency` *and* `peerDependencies`? Rationale: https://github.com/vikejs/vike/pull/2070. Making `vike-react-redux` even more zero config :sparkles: